docs/recipes/authentication.md

```diff
@@ -22,38 +22,52 @@ export class Logout {
 ```
 
 In our state model, we want to track our token and the username. The token
-represents a JWT token that was issued for the session.
+represents a JWT token that was issued for the session. If the user is not authorized,
+the state value is `null`, which corresponds to the semantics. It is very easy to work
+with the `null` value and make the appropriate checks.
```

Comment:
My personal preference is to try to stay away from a state being null because it makes selectors more complex.
In this case the `AuthState.token` selector would now need to check for null;
I would have preferred to check that the token is not null for `AuthState.isAuthenticated`.

Reply:
Okay

Reply:
@markwhitfeld 

Should I change `token` property from optional to `null`? Like:

```ts
defaults: {
  token: null,
  username: null
}
```
Except of plain object `{}`